MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -16,96 +16,116 @@ Existing FOLIO modules ideally conform to the same Values and Criteria.  It is u
 
-# Values
+## How to use this document
+
+When performing a technical evaluation of a module, create a copy of this document.  The evaluation results should be placed in the [module_evaluations](https://github.com/folio-org/tech-council/tree/master/module_evaluations) directory and should conform to the following naming convention: `{JIRA Key}_YYYY-MM-DD-{module name}.MD`, e.g. `TCR-1_2021-11-17-mod-foo.MD`.  The date here is used to differentiate between initial and potential re-evaluation(s).  It should be the date when the evaluation results file was created.
+
+Replace the title with the name of the module or library.
+
+Remove the Frontend or the Backend section.
+
+Use these conventions to indicate the status of each criterion.
+
+* [x] ACCEPTABLE
+* [x] ~INAPPLICABLE~
+* [ ] UNACCEPTABLE
+  * comments on what was evaluated/not evaluated, why a criterion failed
+
+## Values
 1. Module adheres to Community Code of Conduct
-1. Module license is compatible with the FOLIO Project
-1. Module can be included in existing community build processes
-1. Module has robust testing that can run with existing community testing processes
-1. Module can be deployed in the Community’s reference environments without undue burden
-1. Module is secure
-1. Module is multi-tenant
-1. Module is internationalized
-1. Module meets current accessibility requirements
-1. Module offers a cohesive user experience consistent with the rest of FOLIO
-1. Module has developer and end-user documentation
-1. Module depends ONLY on other modules and infrastructure that are already included in FOLIO
-1. Module has a long-term development and maintenance plan
-1. Module is scalable
-1. Module supports high availability
-1. Modules conforms to FOLIO upgrade mechanisms
-
-# Criteria
-
-## Administrative
-* Listed by the Product Council on [Functionality Evaluated by the PC](https://wiki.folio.org/display/PC/Functionality+Evaluated+by+the+PC) with a positive evaluation result.
-
-## Shared/Common
-* Uses Apache 2.0 license (2)
-* Module build MUST produce a valid module descriptor (3, 5)
+2. Module license is compatible with the FOLIO Project
+3. Module can be included in existing community build processes
+4. Module has robust testing that can run with existing community testing processes
+5. Module can be deployed in the Community’s reference environments without undue burden
+6. Module is secure
+7. Module is multi-tenant
+8. Module is internationalized
+9. Module meets current accessibility requirements
+10. Module offers a cohesive user experience consistent with the rest of FOLIO
+11. Module has developer and end-user documentation
+12. Module depends ONLY on other modules and infrastructure that are already included in FOLIO
+13. Module has a long-term development and maintenance plan
+14. Module is scalable
+15. Module supports high availability
+16. Modules conforms to FOLIO upgrade mechanisms
```

Comment:
Better to leave these all as `1.` so it's easier to add/remove as needed without changing every other line?

Reply:
That's fair.